packages/esm-patient-attachments-app/src/attachments/image-preview.component.tsx

```diff
@@ -4,27 +4,27 @@ import { UserHasAccess } from '@openmrs/esm-framework';
 import { Button, ButtonSet, TextInput } from 'carbon-components-react';
 import styles from './image-preview.scss';
 
-interface ImagePreviewProps {
+interface ImageOrPdfPreviewProps {
   content: string;
   collectCaption: boolean;
-  onSaveImage?(dataUri: string, caption: string): void;
+  onSaveImageOrPdf?(dataUri: string, caption: string): void;
```

Comment:
```suggestion
  onSaveFile?(dataUri: string, caption: string): void;
```

Reply:
@ibacher, I believe I have covered all the proposed changes and am able to attach the `pdf file` as per the ticket description.